control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -619,15 +619,15 @@ impl ResizeReplica {
     pub fn new(
         node: &NodeId,
         pool_id: &PoolId,
-        name: &ReplicaName,
+        name: Option<&ReplicaName>,
```

Comment:
Why does it need to be optional?

Reply:
> Why does it need to be optional?

Primarily following other types here. replica uuid is there as mandatory field in most, and name is optional.